### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by cp-shruti-s**

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+      - name: Run prettier
+        run: npx prettier -c .
+
+      - name: Build application
+        run: npm run build
+
+      - name: Craete zip file
+        run: |
+          apt-get update && apt-get install -y zip sed
```
Comment (cp-sumi-k) [inline .github/workflows/deploy.yml:33]: No need to install sed

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+      - name: Artifacts zip file
+        uses: actions/upload-artifact@v3
+        with:
+          name: dist-with
+          path: canopas_blog.zip
+
+      - name: Upload to S3
+        run: |
+          - S3_FILE_NAME="canopas_blog_${CI_PIPELINE_IID}.zip"
```
Comment (cp-sumi-k) [inline .github/workflows/deploy.yml]: Is `CI_PIPELINE_IID` available on github actions? Search on it

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

File: .github/workflows/deploy.yml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+      - name: Build application
+        run: npm run build
+
+      - name: Craete zip file
+        run: |
+          apt-get update && apt-get install -y zip sed
+          npm run build
+          zip canopas_blog.zip ./dist/
+
+      - name: Artifacts zip file
```
Comment (cp-sumi-k) [inline .github/workflows/deploy.yml:37]: No need of this step, create only one step for zip and upload file

File: .github/workflows/deploy.yml (hunk 2 above)
Comment (cp-shruti-s) [reply .github/workflows/deploy.yml]: no, but [GITHUB_RUN_ID](https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables) is available

Comment (cp-sumi-k) [reply .github/workflows/deploy.yml]: Have to use it instead of `CI_PIPELINE_IID`

**Review by cp-sumi-k — APPROVED** [review]

**Pull request merged**